controllers/user.controller.js

```diff
@@ -5,6 +5,58 @@ const fs = require('fs');
 const path = require('path');
 const Jimp = require('jimp');
 const gravatar = require('gravatar');
+const mg = require('../mailgunConfig');
```

Comment:
Umieszczaj importy z bibliotek zewnętrznych na początku, a następnie lokalne importy. Oddzielaj je pustymi liniami.
Sortuj importy alfabetycznie w obrębie każdej grupy. Ułatwia to znalezienie potrzebnych importów.